tests/expectations/compiler/constructor/constructor_without_async_fail.out

```diff
@@ -1,4 +1,4 @@
-Error [EPAR0370005]: expected 'const', 'struct', 'record', 'mapping', '@', 'async', 'function', 'transition', 'inline', 'script', '}' -- found 'constructor'
+Error [EPAR0370005]: expected '(', '@', 'async', 'function', 'inline', 'script', 'transition' -- found 'constructor'
```

Comment:
Does this error message need to include `const`, `struct`, `record`, and `mapping`?